.github/workflows/tiledb-cloud-csharp.yml

```diff
@@ -0,0 +1,104 @@
+name: TileDB-Cloud-CSharp
+
+on:
+  push:
+    tags: [ '*' ]
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  Pack-TileDB-Cloud:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+      # Checks out repository
+      - uses: actions/checkout@v3
+
+      - name: Remove existing .NET versions
+        shell: bash
+        run: |
+          rm -rf $DOTNET_ROOT
```

Comment:
Why do we need to remove the existing version?

Reply:
I did this to limit the runners to a specific .NET version, since some GH runners have [.NET SDKs preinstalled](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#net-core-sdk). I think it's good to limit the environment to only the .NET SDK we are targeting in the test to prevent .NET from implicitly selecting an available SDK to build / run with. I can remove it if you'd like, I just included it as a precaution - I didn't want tests for .NET6 passing only because the SDK automatically selected .NET5 in the background. (or vice-versa)

Reply:
No that's fine, but please add a (short) comment inline about why in case someone else wonders in the future.